.github/workflows/cicd.yaml

```diff
@@ -101,6 +101,7 @@ jobs:
           PGDATABASE: postgis
           APP_HOST: 0.0.0.0
           APP_PORT: 8080
+          ENABLED_EXTENSIONS: transaction,bulk_transactions,query,sort,fields,filter,pagination,collection_search
```

Comment:
could this be set in `conftest.py` so we can run this locally?

Reply:
in the contests if possible

Reply:
this change was for `scripts/validate` but I added the same line to the application in  `docker-compose.yml` so the transaction extensions are enabled there.